src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.test.tsx

```diff
@@ -47,13 +47,16 @@ describe('<BrowserTrackConfig />', () => {
   };
 
   describe('behaviour', () => {
-    it('sets all tracks to be updated when the all tracks checkbox is selected', () => {
+    it('sets all tracks to be updated when the all tracks is selected', () => {
```

Comment:
there's a problem with `the all tracks is` 🙂 

Maybe rephrase the whole test description to `it('can update all tracks', () => {...})`?